Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/main/MainActivityTest.kt

```diff
@@ -0,0 +1,429 @@
+        captureHomeFragment("submission_done")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestError() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_ERROR_ITEM
+        )
+        captureHomeFragment("test_error")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestFetching() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_FETCHING_ITEM
+        )
+        captureHomeFragment("test_fetching")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestInvalid() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_INVALID_ITEM
+        )
+        captureHomeFragment("test_invalid")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestNegative() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_NEGATIVE_ITEM
+        )
+        captureHomeFragment("test_negative")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestPositive() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_POSITIVE_ITEM
+        )
+        captureHomeFragment("test_positive")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentTestPending() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(
+            submissionTestResultItem = HomeData.Submission.TEST_PENDING_ITEM
+        )
+        captureHomeFragment("test_pending")
+    }
+
+    @Screenshot
+    @Test
+    fun captureHomeFragmentStatistics() {
+        every { homeFragmentViewModel.homeItems } returns homeFragmentItemsLiveData(HomeData.Tracing.LOW_RISK_ITEM)
+        launchActivity<MainActivity>()
+        onView(withId(R.id.recycler_view)).perform(recyclerScrollTo(3))
+        Statistics.statisticsData?.items?.forEachIndexed { index, _ ->
+            onView(withId(R.id.statistics_recyclerview)).perform(recyclerScrollTo(index))
+            takeScreenshot<HomeFragment>("statistics_card_$index")
+        }
+    }
+
+    @Screenshot
+    @Test
+    fun captureContactDiaryOverviewFragment() {
+        every { contactDiaryOverviewViewModel.listItems } returns contactDiaryOverviewItemLiveData()
+        launchActivity<MainActivity>()
+        onView(withId(R.id.main_bottom_navigation))
+            .perform(selectBottomNavTab(R.id.contact_diary_nav_graph))
+        takeScreenshot<ContactDiaryOverviewFragment>()
+
+        onView(withId(R.id.contact_diary_overview_recyclerview))
+            .perform(recyclerScrollTo(1))
+        takeScreenshot<ContactDiaryOverviewFragment>("2", delay = 0L)
```

Comment:
I guess you need to keep default delay here, the first and the second screenshots look the same